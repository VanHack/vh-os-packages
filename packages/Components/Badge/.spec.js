import React from 'react';
import VHBadge from '.';
import { render, renderer } from '../../../setupTests';

describe('[Component] - VHBadge', () => {
  it('renders component without crashing', () => {
    const { getByText } = render(<VHBadge title="simple badge" />);
    expect(getByText('simple badge')).toMatchSnapshot();
  });

  it('renders VHBadge with specific color', () => {
    const json = renderer(<VHBadge title="badge" />).toJSON();
    expect(json).toHaveStyleRule('border', '1px solid #0675ce');
    expect(json).toHaveStyleRule('color', '#0675ce');
  });
});
