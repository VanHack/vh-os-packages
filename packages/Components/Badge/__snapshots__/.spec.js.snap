// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] - VHBadge renders component without crashing 1`] = `
.c0 {
  border: 1px solid #0675ce;
  color: #0675ce;
  padding: 5px;
  border-radius: 4px;
  font-size: 12px;
}

<div
  class="c0"
>
  simple badge
</div>
`;
